draft-e2ee.md

```diff
@@ -183,7 +183,7 @@ The features and challenges listed below are framed comprehensively rather than
 Features
```

Comment:
I think it is better to keep this section as it is now. There is some subtly on the language we chose ;)